jenkins-master/jobs/scripts/workspace/runtests.py

```diff
@@ -136,6 +136,10 @@ def query_args(self):
 
         return args
 
+class MediaRunner(BaseRunner):
+
+    def __init__(self, *args, **kwargs):
+        BaseRunner.__init__(self, *args, **kwargs)
```

Comment:
And it should be a super(MediaRunner) call instead of a BaseRunner call. But all of call BaseRunnder.__init__.